src/main/ipv-stub/service/dynamodb-form-response-service.ts

```diff
@@ -1,3 +1,72 @@
 import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocument } from "@aws-sdk/lib-dynamodb";
+
+import { UserIdentity } from "../types";
 
 const client = new DynamoDBClient({});
+const dynamo = DynamoDBDocument.from(client);
+
+const tableName = `${process.env.ENVIRONMENT}-UserIdentity`;
+
+export const getUserIdentityWithAuthCode = async (
+  authCode: string
+): Promise<UserIdentity> => {
+  const userIdentity = await dynamo.get({
+    TableName: tableName,
+    Key: { UserIdentityId: authCode },
+  });
+  return userIdentity as unknown as UserIdentity;
+};
+
+export const putUserIdentityWithAuthCode = async (
+  authCode: string,
+  userIdentity: UserIdentity
+) => {
+  return await dynamo.put({
+    TableName: tableName,
+    Item: {
+      UserIdentityId: authCode,
+      userIdentity,
+    },
+  });
+};
+
+export const deleteUserIdentityWithAuthCode = async (authCode: string) => {
```

Comment:
Any reason this doesn't return `Promise<UserIdentity` like the others?

Reply:
Not relevant anymore as method has been removed.